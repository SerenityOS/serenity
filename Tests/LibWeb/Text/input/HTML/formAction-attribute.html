<!DOCTYPE html>
<script src="../include.js"></script>
<button formaction="http://www.example.com/"></button>
<input formaction="http://www.example.com/">
<script>
    test(() => {
        const elementNames = [
            "button",
            "input",
        ];
        for (const elementName of elementNames) {
            const element = document.querySelector(elementName);
            println(`${elementName}.formAction initial value: ${element.formAction}`);
            element.formAction = "";
            println(`Final segment of ${elementName}.formAction after setting to the empty string: ${element.formAction.split('/').pop()}`);
            element.formAction = "../test.html";
            println(`Final segment of ${elementName}.formAction after setting to "../test.html": ${element.formAction.split('/').pop()}`);
            element.remove();
        }
    });
</script>
