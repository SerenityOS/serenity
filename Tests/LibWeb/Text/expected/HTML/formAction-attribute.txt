button.formAction initial value: http://www.example.com/
Final segment of button.formAction after setting to the empty string: formAction-attribute.html
Final segment of button.formAction after setting to "../test.html": test.html
input.formAction initial value: http://www.example.com/
Final segment of input.formAction after setting to the empty string: formAction-attribute.html
Final segment of input.formAction after setting to "../test.html": test.html
